cli/src/main/java/org/jboss/pnc/bacon/cli/App.java

```diff
@@ -152,7 +150,7 @@ public int run(String[] args) throws Exception {
                 }
 
                 // signal that an error has occurred
-                throw new FatalException();
+                throw new FatalException("Unknown RuntimeException", ex);
```

Comment:
This file randomly seems to use both variable names `e` and `ex` for exceptions. Could we standardize on `e`?

Reply:
I've pushed some changes ; this is mostly historic to be honest